blog/ga-patchfix-mars.2.adoc

```diff
@@ -0,0 +1,39 @@
+= Patch fix for Red Hat JBoss Developer Studio 9.0.0.GA and JBoss Tools 4.3.0.Final
```

Comment:
That's invalid according to the branding rules.

How about "EGit fix for Red Hat JBoss Developer Studio 9.0.0.GA and JBoss Tools 4.3.0.Final update sites" ?